nistats/first_level_model.py

```diff
@@ -246,10 +250,12 @@ def __init__(self, t_r=None, slice_time_ref=0., hrf_model='glover',
                  fir_delays=[0], min_onset=-24, mask=None, target_affine=None,
                  target_shape=None, smoothing_fwhm=None, memory=Memory(None),
                  memory_level=1, standardize=False, signal_scaling=0,
-                 noise_model='ar1', verbose=1, n_jobs=1,
+                 noise_model='ar1', verbose=0, n_jobs=1,
                  minimize_memory=True):
         # design matrix parameters
         self.t_r = t_r
+        if t_r > 10:
```

Comment:
I thought it could be a very nice feature, its an inexpensive check for a very common mistake handling the time unit of t_r, moreover the impact of this type of mistake is very important.